tests/test_encoding.nim

```diff
@@ -0,0 +1,125 @@
+import std/options
+import pkg/unittest2
+import pkg/chronos
+import pkg/stint
+import pkg/nimcrypto
+import ../web3
+import ./test_utils
+import ./encodingcontract
```

Comment:
Nit pick here, nothing mandatory.

The rest of the repo using a different import structure, so maybe we should keep it consistent?

Reply:
Good idea, consistency is important. I changed it with the latest commit.